app/models/harvest.rb

```diff
@@ -7,6 +7,7 @@ class Harvest < ActiveRecord::Base
   belongs_to :crop
```

Comment:
Later we could say "you're harvesting something you never planted. Would you like to create a planting?"